src/components/pages/Navbar/Navbar.js

```diff
@@ -43,6 +45,21 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isAuthenticated]);
 
+  const profile_id = user.profile_id;
+  const isMentor = user.role_id === 3;
```

Comment:
Nice concise way of getting to the boolean here and avoiding a needless ternary!